src/Arcus.Templates.AzureFunctions.ServiceBus.Queue/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM mcr.microsoft.com/azure-functions/dotnet:4 AS base
 WORKDIR /app
```

Comment:
Is this still the correct base image for runtime ?

Reply:
Will be looked over in a follow-up PR.